candies/cli/cli.py

```diff
@@ -57,6 +61,60 @@ def help(self, *args, **kwargs):
         """
         return self.main.help(*args, **kwargs)
 
+    def parse(self, callable: Union[Callable[[str], Any], Type]) \
+            -> Union[Type, Callable[[str], Any]]:
+        """Defines a parser for a custom type.
+
+        Args:
+            callable: A callable that converts a string to an instance
+                of a custom type. For example, a parser function or a
+                type that accepts a string to a constructor.
```

Comment:
`to a constructor` -> `in the constructor`?

Reply:
Removed this sentence.